org.jacoco.agent.rt.test/src/org/jacoco/agent/rt/internal/output/TcpConnectionTest.java

```diff
@@ -217,8 +217,7 @@ private void readAndAssertData() throws IOException {
 
 	@Test
 	public void testRemoteReset() throws Exception {
-		data.getExecutionData(Long.valueOf(123), "Foo", 1)
-				.getProbes()[0] = true;
+		data.getExecutionData(Long.valueOf(123), "Foo", 1).getProbes()[0] = 1;
```

Comment:
Yes, that is all due to spotless indeed. In this case the line length is decreased by 3 characters and now it decides to fit on a single line. Otherwise the build will fail :disappointed: We can also decide to disable spotless and indeed minimize the diff.